index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 class='header'>Hello World, we are software developers!</h1>
```

Comment:
- [ ] Always make sure to include an explanation, a "why", for each requested change, and use polite words like kindly and It would be better. Kindly Implement it.

Reply:
Kindly include the <h1> tag within a section. It would be easy to deal with the styling in the future when you do so 😃